v2/snmp/serverhooks.go

```diff
@@ -18,10 +18,10 @@ type ServerHooks struct {
 	// Error is called after an error condition has been detected.
 	Error func(config *serverConfig, err error)
 
-	// WriteComplete is called after a packet has been written
+	// WriteDone is called after a packet has been written
```

Comment:
comment wrong

Reply:
Reverted unintended changes to serverhooks.

Reply:
Reverted unintended changes to serverhooks.